runner/rootless-startup.sh

```diff
@@ -18,6 +18,10 @@ fi
 if [ -n "${DOCKER_REGISTRY_MIRROR}" ]; then
 jq ".\"registry-mirrors\"[0] = \"${DOCKER_REGISTRY_MIRROR}\"" /home/runner/.config/docker/daemon.json > /tmp/.daemon.json && mv /tmp/.daemon.json /home/runner/.config/docker/daemon.json
 fi
+
+if [ -n "${DOCKER_DEFAULT_NETWORK_CIDR_RANGE}" ]; then
```

Comment:
Hey! Thanks a lot for your contribution.
A quick comment on your awesome work, sorry for being nit-picky but, is that R in CIDR stands for Range already?
Then, can we instead name this after the setting key path, so that it would look like `DOCKER_DEFAULT_ADDRESS_POOL_BASE` paired with `DOCKER_DEFAULT_ADDRESS_POOL_SIZE`?
Naming things that way will also enable us to make those consistent with the existing variable name like `DOCKER_REGISTRY_MIRROR`. WDYT?

Reply:
It makes a lot of sense. I didn't think too much about the naming (too focused on resolving the problem I had) and will change it.

Reply:
Oops! Sorry about that. I somehow misunderstood that for years 😂 Thank you so much for pointing it out! You saved me.